apps/fxc-front/src/app/components/2d/path-element.ts

```diff
@@ -258,7 +258,7 @@ export class PathElement extends connect(store)(LitElement) {
 
     if (result.closingPoints) {
       this.closingSector.center = result.closingPoints.in;
-      this.closingSector.radius = score.closingRadiusM;
+      this.closingSector.radius = score.closingRadiusM/1000;
```

Comment:
you should install the ESLint plugin in VSCode to format the code on save

Reply:
Forgot to activate it.